tests/core/test_test.py

```diff
@@ -123,6 +124,41 @@ async def test_testing_warns_if_action_unknown(
     assert "which is not part of the training data / domain" in output
 
 
+async def test_testing_with_utilizing_retrieval_intents(
+    intent_retrieval_agent: Agent,
+    intent_retrieval_bot_test_stories: Path,
+    intent_retrieval_bot_test_results: Path,
+):
+    if not os.path.exists(intent_retrieval_bot_test_results):
```

Comment:
small hack as it's a `Path`
```suggestion
    if not intent_retrieval_bot_test_results.exists():
```